sdcm/cluster.py

```diff
@@ -1908,7 +1908,8 @@ def upgrade_manager_agent(self, scylla_mgmt_address: str, start_agent_after_upgr
             self.remoter.sudo("zypper update scylla-manager-agent -y")
         else:
             self.remoter.sudo("apt-get update", ignore_status=True)
-            self.remoter.sudo("apt-get install -o DPkg::Lock::Timeout=300 -y scylla-manager-agent")
+            self.remoter.sudo("apt-get install -o Dpkg::Lock::Timeout=300 -o Dpkg::Options::='--force-confold' "
```

Comment:
usually we add also `DEBIAN_FRONTEND=noninteractive ` before `apt-get`

Reply:
fixed